{"uid":"678cde50af59e9d","name":"test_modern_endpoint_with_multiple_related_endpoints","fullName":"tests.integration.relations.test_relations#test_modern_endpoint_with_multiple_related_endpoints","historyId":"3e609d30cce71f7c1f78998935689eed","time":{"start":1721698438926,"stop":1721698537371,"duration":98445},"status":"failed","statusMessage":"Failed: DID NOT RAISE <class 'psycopg2.OperationalError'>","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/relations/test_relations.py\", line 142, in test_modern_endpoint_with_multiple_related_endpoints\n    for attempt in Retrying(stop=stop_after_delay(60 * 5), wait=wait_fixed(10)):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 443, in __iter__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 376, in iter\n    result = action(retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 398, in <lambda>\n    self._add_action_func(lambda rs: rs.outcome.result())\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 451, in result\n    return self.__get_result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 403, in __get_result\n    raise self._exception\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/relations/test_relations.py\", line 144, in test_modern_endpoint_with_multiple_related_endpoints\n    with pytest.raises(psycopg2.OperationalError):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python_api.py\", line 997, in __exit__\n    fail(self.message)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 177, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\nFailed: DID NOT RAISE <class 'psycopg2.OperationalError'>","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1721698118160,"stop":1721698118161,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1721698118161,"stop":1721698118312,"duration":151},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1721698118161,"stop":1721698118161,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1721698438925,"stop":1721698438925,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"failed","statusMessage":"Failed: DID NOT RAISE <class 'psycopg2.OperationalError'>","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/relations/test_relations.py\", line 142, in test_modern_endpoint_with_multiple_related_endpoints\n    for attempt in Retrying(stop=stop_after_delay(60 * 5), wait=wait_fixed(10)):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 443, in __iter__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 376, in iter\n    result = action(retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 398, in <lambda>\n    self._add_action_func(lambda rs: rs.outcome.result())\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 451, in result\n    return self.__get_result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 403, in __get_result\n    raise self._exception\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/relations/test_relations.py\", line 144, in test_modern_endpoint_with_multiple_related_endpoints\n    with pytest.raises(psycopg2.OperationalError):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python_api.py\", line 997, in __exit__\n    fail(self.message)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 177, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\nFailed: DID NOT RAISE <class 'psycopg2.OperationalError'>","steps":[],"attachments":[{"uid":"6dfb49caa628895d","name":"log","source":"6dfb49caa628895d.txt","type":"text/plain","size":5137}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1721698537375,"stop":1721698537375,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1721698537376,"stop":1721698537869,"duration":493},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1721698537869,"stop":1721698537869,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1721698537870,"stop":1721698537870,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1721698537870,"stop":1721698537870,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.relations"},{"name":"suite","value":"test_relations"},{"name":"host","value":"fv-az774-16"},{"name":"thread","value":"7447-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.relations.test_relations"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"d26eea98a49190ab","status":"unknown","time":{}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":2,"broken":0,"skipped":0,"passed":41,"unknown":2,"total":45},"items":[{"uid":"5f87dea82e7df3cd","reportUrl":"../119//#testresult/5f87dea82e7df3cd","status":"failed","statusDetails":"Failed: DID NOT RAISE <class 'psycopg2.OperationalError'>","time":{"start":1721612730335,"stop":1721612832173,"duration":101838}},{"uid":"5c0fc89e25f9c59","reportUrl":"../118//#testresult/5c0fc89e25f9c59","status":"passed","time":{"start":1721527047755,"stop":1721527094662,"duration":46907}},{"uid":"50dbd141cc7fabe7","reportUrl":"../117//#testresult/50dbd141cc7fabe7","status":"passed","time":{"start":1721440022904,"stop":1721440066956,"duration":44052}},{"uid":"65e84f95d0ce2a1b","reportUrl":"../113//#testresult/65e84f95d0ce2a1b","status":"passed","time":{"start":1721266646756,"stop":1721266730090,"duration":83334}},{"uid":"2e6aa89f44afb8e1","reportUrl":"../111//#testresult/2e6aa89f44afb8e1","status":"passed","time":{"start":1721180390698,"stop":1721180474165,"duration":83467}},{"uid":"a4cd495580e18f38","reportUrl":"../110//#testresult/a4cd495580e18f38","status":"passed","time":{"start":1721093898291,"stop":1721093941968,"duration":43677}},{"uid":"a5a1ba6d1b8a4a34","reportUrl":"../109//#testresult/a5a1ba6d1b8a4a34","status":"unknown","time":{}},{"uid":"4db948096dba0774","reportUrl":"../108//#testresult/4db948096dba0774","status":"passed","time":{"start":1720921214572,"stop":1720921300040,"duration":85468}},{"uid":"b3169a9110f9237d","reportUrl":"../107//#testresult/b3169a9110f9237d","status":"passed","time":{"start":1720835143684,"stop":1720835187197,"duration":43513}},{"uid":"f8b1acdd2b1f79ad","reportUrl":"../106//#testresult/f8b1acdd2b1f79ad","status":"passed","time":{"start":1720748065466,"stop":1720748149981,"duration":84515}},{"uid":"1dc2adbdfb6032f","reportUrl":"../105//#testresult/1dc2adbdfb6032f","status":"passed","time":{"start":1720662150719,"stop":1720662231939,"duration":81220}},{"uid":"3ad1b0d97216ef57","reportUrl":"../104//#testresult/3ad1b0d97216ef57","status":"passed","time":{"start":1720576124051,"stop":1720576208820,"duration":84769}},{"uid":"4f51bccbbb2f142a","reportUrl":"../103//#testresult/4f51bccbbb2f142a","status":"passed","time":{"start":1720489210872,"stop":1720489292005,"duration":81133}},{"uid":"68eb9563268b4a5","reportUrl":"../102//#testresult/68eb9563268b4a5","status":"passed","time":{"start":1720402370938,"stop":1720402453667,"duration":82729}},{"uid":"3b17475cde441322","reportUrl":"../101//#testresult/3b17475cde441322","status":"passed","time":{"start":1720316637773,"stop":1720316728681,"duration":90908}},{"uid":"1630eae31ba39d2b","reportUrl":"../100//#testresult/1630eae31ba39d2b","status":"passed","time":{"start":1720229637229,"stop":1720229720705,"duration":83476}},{"uid":"c66d44083f3ecdaa","reportUrl":"../99//#testresult/c66d44083f3ecdaa","status":"passed","time":{"start":1720143937672,"stop":1720144023042,"duration":85370}},{"uid":"51e1704927804465","reportUrl":"../98//#testresult/51e1704927804465","status":"passed","time":{"start":1720057071064,"stop":1720057151744,"duration":80680}},{"uid":"ed4ed6a4c7fe1696","reportUrl":"../97//#testresult/ed4ed6a4c7fe1696","status":"passed","time":{"start":1719970376475,"stop":1719970458820,"duration":82345}},{"uid":"35b304bcf8c654a0","reportUrl":"../96//#testresult/35b304bcf8c654a0","status":"passed","time":{"start":1719883952122,"stop":1719884041751,"duration":89629}}]},"tags":["asyncio","group(1)"]},"source":"678cde50af59e9d.json","parameterValues":[]}